libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -799,6 +796,8 @@ void AP_InertialSensor_Backend::update_gyro(uint8_t instance) /* front end */
         }
     }
 #endif
+
+    set_primary_gyro(_imu._primary_gyro);
```

Comment:
This function dons't do anything? If its a backed should it pass a bool "is_primary" rather than the instance?

Reply:
Right, this is to give the backends an opportunity to do something - that something is the more controversial bit, so just putting the framework in place for now.

bool is_primary would also work if you think that's better

Reply:
Fixed